tasks/patch_server.rb

```diff
@@ -259,40 +260,51 @@ def reboot_required(family, release)
 # There are no updates available, exit cleanly rebooting if the override flag is set
 if updatecount.zero?
   if reboot_override == true
-    log.info 'Rebooting'
+    log.error 'Rebooting'
     output('Success', reboot, security_only, 'No patches to apply, reboot triggered', '', '', '', pinned_pkgs, starttime)
     $stdout.flush
-    log.info 'No patches to apply, rebooting as requested'
+    log.error 'No patches to apply, rebooting as requested'
     p1 = fork { system('nohup /sbin/shutdown -r +1 2>/dev/null 1>/dev/null &') }
     Process.detach(p1)
   else
     output('Success', reboot, security_only, 'No patches to apply', '', '', '', pinned_pkgs, starttime)
-    log.info 'No patches to apply, exiting'
+    log.error 'No patches to apply, exiting'
```

Comment:
Should these be errors? Maybe debug?

Reply:
This was a miss on my part.  I was trying to figure out why nothing was working when I changed things but forgot to use `environment=` on the task command line.  Ramped up all of the debugging to try and figure it out, then clue batted myself.

Fixed on the branch, will be in the next release